README.md

```diff
@@ -130,11 +130,22 @@ Example: `partners_count: 4`
 Fractions of the original dataset each partner receives to mock a collaborative ML scenario where each partner provides data for the ML training.  
 Example: `amounts_per_partner: [0.3, 0.3, 0.1, 0.3]`
 
-`samples_split_option`: `random` or `stratified`  
+`samples_split_option`: `random` (default) or `stratified`  
 How the original dataset data samples are split among partners:
 
-- `random`: the dataset is shuffled and the split is done randomly, ensuring a homogeneous data distribution among partners
-- `stratified`: the dataset is stratified per labels, so that partners have datasets covering different regions of space
+- `random`: the dataset is shuffled and partners receive data samples selected randomly
+- `stratified`: the dataset is stratified per labels and each partner receives certain classes only
+
+`advanced_split`: `[[nb of clusters (int), 'shared' or 'specific']]`  
+In certain cases it might be interesting to split the dataset among partners in a more elaborate way. For that we consider the data samples from the initial dataset as split in clusters, defined by their label/class.
+It is possible to configure the `advanced_split` parameter by indicating, for each partner in sequence, the following 2 elements:
+
+- `nb of clusters (int)`: the given partner will receive data samples covering this number of different clusters (clusters of data samples per labels/classes) 
+- `'shared'` or `'specific'`:
+  - `'shared'`: all partners with option `'shared'` receive data samples picked from clusters they share data samples from
+  - `'specific'`: each partner with option `'specific'` receives data samples picked from cluster(s) it is the only one to receive from
+
+Example: `[[7, 'shared'], [6, 'shared'], [2, 'specific'], [1, 'specific']]`
```

Comment:
I would use tuple here:
`[(7, 'shared'), (6, 'shared'), (2, 'specific'), (1, 'specific')]`
https://stackoverflow.com/questions/1708510/list-vs-tuple-when-to-use-each

Reply:
I know, but the yaml parser used for the config file doesn't pick it up that way.